example-get-started/code/src/evaluate.py

```diff
@@ -100,7 +100,7 @@ def main():
         test, _ = pickle.load(fd)
 
     # Evaluate train and test datasets.
-    with Live(EVAL_PATH, cache_images=True, dvcyaml=False) as live:
+    with Live(EVAL_PATH, dvcyaml=False, report=None) as live:
```

Comment:
Very minor, but we could probably drop `report=None` since it at worst will be an extra file and `report=None` will be the default in dvclive 3.0.

Reply:
yes, I'm following that ticket . I can change it when it's merged